### PR TITLE
Laravel 8 upgrade

### PR DESCRIPTION
**Pull request by wreality**

### CODE REVIEW

File: backend/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# For more information: https://laravel.com/docs/sail
```
Comment (gmeben) [inline backend/docker-compose.yml:1]: I was under the impression that Lando would've eliminated the need for this file.

File: backend/app/Models/User.php [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+{
+    use HasFactory, Notifiable;
+
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array
+     */
+    protected $fillable = [
+        'name',
```
Comment (gmeben) [inline backend/app/Models/User.php:20]: This should probably be called `username` right?

File: backend/docker-compose.yml (hunk 1 above)
Comment (wreality) [reply backend/docker-compose.yml:1]: That docker compose comes with the Laravel project. I'm leaving it there for the time being as a reference until we decide how our production containers will be defined.

File: backend/app/Models/User.php (hunk 2 above)
Comment (wreality) [reply backend/app/Models/User.php:20]: Defaults from Laravel. I was going to make these changes in a separate PR. You're right though, I'll update it to match how things worked before the upgrade.

**Review by gmeben — APPROVED** [review]
I got this up and running locally without issue. Everything looks good as far as I'm concerned.

**Pull request merged**